.gitignore

```diff
@@ -13,6 +13,7 @@ rpm-build
 # Eclipse/PyDev stuff...
 .project
 .pydevproject
+.settings
```

Comment:
This change should be made in a separate PR since it's not directly related to the rest of the changes here.